recipe/bld.bat

```diff
@@ -1,4 +1,50 @@
-robocopy . "%PREFIX%" /S /XF bld.bat
-if errorlevel NEQ 1 exit 1
+set "PREFIX_NSIS=%PREFIX%\NSIS"
+robocopy . "%PREFIX_NSIS%" /S /XF bld.bat Docs Examples
+if errorlevel 1 exit 1
```

Comment:
Hmm...for some reason this when sideways. Guessing we need an `/XF` before each of these?